src/Interfaces/ILtiServiceConnector.php

```diff
@@ -26,5 +27,11 @@ public function getAll(
         string $key
     ): array;
 
+    public function get(
+        ILtiRegistration $registration,
+        array $scopes,
+        IServiceRequest $request
+    ): LtiLineitem;
```

Comment:
Think of the `LtiServiceConnector` as an HTTP client. Various services (AGS, NRPS, etc.) defined the in the LTI spec use this client to make calls to the LMS. Specifying the `LtiLineItem` return type here too closely ties the implementation of this generic client to the AGS (`AssignmentGradesService`... i.e. grades). What if NRPS (`NamesRolesProvisioningService`... i.e. rosters) needed to make a GET request? It would be able to use this method because of the return type.